spec/factories/users.rb

```diff
@@ -34,4 +34,10 @@
   factory :placements_user, class: "Placements::User", parent: :user do
     service { "placements" }
   end
+
+  factory :placements_support_user,
+          class: "Placements::SupportUser",
+          parent: :user do
+    service { "placements" }
+  end
```

Comment:
If we do stick with `Placements::SupportUser` being its own entity, then we probably want to add:

```ruby
support_user { true }
```

to make it more true to reality.

Reply:
Because `Placements::SupportUser` is scoped to `support_user == true`, this isn't necessary. I might be misunderstanding...

Reply:
I'd assumed `default_scope` only filtered 'find' queries, so it wasn't possible to find a non-support user from the database. I didn't realise it also automatically sets `support_user = true` in the model.

```ruby
Placements::User.new.attributes["support_user"] # => false
Placements::SupportUser.new.attributes["support_user"] # => true
```

Very cool! 🤓 Every day's a school day!